README.md

```diff
@@ -1 +1,18 @@
-# video_pong
+# Video Pong
+
+Video Pong is a digital take on classic table tennis. This was created with ES6 JavaScript and the HTML5 Canvas API within a React framework.  
+
+## Table of Contents
+
+- [Video Pong](#video-pong)
+  - [Table of Contents](#table-of-contents)
+  - [How do I get started?](#how-do-i-get-started)
+
+## How do I get started?
+
+1. Clone down this repo by running `git clone git@github.com:Codo-Baggins/video_pong.git` in your terminal 
+2. `cd` into the cloned down directory
+3. Run `npm start`
```

Comment:
need more detail here.

i.e. 
`npm install` step
"built with" section highlighting notable package dependencies
other common readme items.

We should include documentation as a part of our "Definition of Done". To close a story/issue, code should be tested, documents, etc.